python/activator/middleware_interface.py

```diff
@@ -327,12 +322,22 @@ def __init__(self, central_butler: Butler, image_bucket: str, visit: FannedOutVi
                  skymap: str, local_repo: str, local_cache: DatasetCache,
                  prefix: str = "s3://"):
         self.visit = visit
+        # Usually prompt processing only cares about on-sky images and expects all of
+        # them to have equatorial coordinates and sky rotation angles. For other images,
+        # ISR can be done but sensible results from further pipelines cannot be expected.
+        # Continue the processing but using the coordinates to preload spatial datasets
+        # will be skipped. This allows cases where only ISR is wanted.
+        self._skip_spatial_preload = False
         if self.visit.coordinateSystem != FannedOutVisit.CoordSys.ICRS:
-            raise ValueError("Only ICRS coordinates are supported in Visit, "
-                             f"got {self.visit.coordinateSystem!r} instead.")
+            self._skip_spatial_preload = True
+            _log.error("Only ICRS coordinates are fully supported. "
+                       f"Got {self.visit.coordinateSystem!r} instead in {self.visit}. "
+                       "Spatial datasets won't be loaded.")
```

Comment:
Should this be a warning, given that we _are_ plowing ahead anyway?

Reply:
I slightly prefer this to be an error for now, so (1) it does not drown in so many warnings from the pipelines, and (2) in a way it is an error as the system isn't really designed to process those visits and something may go wrong later.  I feel that this may be similar to the level of the empty qgraph when there are no templates but we continue with the next best thing.